lib/getstats.js

```diff
@@ -429,7 +446,7 @@ function standardizeChromeStats(response) {
 
   var framesPerSecond = getStatValue('framesPerSecond');
   if (typeof framesPerSecond === 'number') {
-    standardizedStats.frameRateInput = framesPerSecond;
+    standardizedStats.frameRateSent = framesPerSecond;
```

Comment:
@syerrapragada Is it also possible to determine `frameRateReceived`?

Reply:
@manjeshbhargav frameRateReceived is not available in stats.